spec/support/proposal_spec_helper.rb

```diff
@@ -0,0 +1,10 @@
+module ProposalSpecHelper
+  def fully_approve(proposal)
```

Comment:
I guess I just wanted to indicate that it would approve all of the Approvals...totally open to a different name for it, though.